src/constants.js

```diff
@@ -490,6 +490,10 @@ export const styles = {
     width: "75%"
   },
 
+  resultsHeader: { width: "75%", float: "left" },
+
+  resultsHeaderAccuracy: { fontSize: 14, position: "relative", top: 16 },
```

Comment:
nit: do we want to format this on multiple lines for consistency?

Reply:
This is how JsPrettier did the formatting, and occurs elsewhere in this file too, so I think it's okay.